### DISCUSSION TITLE
Mjml appends Multiple times inside the wrapper in React App

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#4960] [General]
![image](https://user-images.githubusercontent.com/119593989/223320349-636447d4-7dcd-49a4-b545-06ac28894c5f.png)
![image](https://user-images.githubusercontent.com/119593989/223320427-59c4a6a9-e774-4bea-afa6-bc4aba181b65.png)

**Comment by artf** [comment]
If you have the StorageManager enabled that's correct behaviour (you're appending new components on top of those already loaded). You can check if the mjml component is already inside and skip the append.
